REST/RESTListener+Replicate.cc

```diff
@@ -190,20 +204,26 @@ namespace litecore { namespace REST {
         }
 
 
-        HTTPStatus wait() {
-            Lock lock(_mutex);
-            _cv.wait(lock, [this]{return finished();});
-            return _finalResult;
-        }
-
-
         void stop() override {
             Lock lock(_mutex);
             if (_repl) {
                 c4log(ListenerLog, kC4LogInfo, "Replicator task #%u stopping...", taskID());
                 _repl->stop();
             }
         }
+        
+
+        void setStopOnIdle(bool stopOnIdle) {
+            _stopOnIdle = stopOnIdle;
+        }
+
+        
+        void setAuth(slice authType, slice user, slice psw) {
+            _authType = authType;
```

Comment:
It doesn't seem like this is going to be of any use, since it looks to assume basic auth (as opposed to token)

Reply:
Maybe keep it for the possible extension in the future?